roles/openshift_health_checker/library/rpm_version.py

```diff
@@ -106,7 +113,9 @@ def _check_pkg_versions(found_pkgs_dict, expected_pkgs_dict):
                 "The following packages were found to be installed with an incorrect version: {}".format('\n'.join([
                     "    \n{}\n    Required version: {}\n    Found versions: {}".format(
                         pkg_name,
-                        pkg["required_version"],
+                        ' or '.join(pkg["required_versions"]) if len(pkg["required_versions"]) < 3
```

Comment:
This code produces "a" for 1 version, "a or b" for 2 versions, and "a, b, c, d, ..., x, y, or z" for 3 or more versions.  I didn't want to use just ", " because the output could be interpreted as a list of required versions.  I didn't want to use " or " because it would look awkward with more than 2 versions.  The only other option I could think of was to change "Required version:" to "Required version: Any of the following:", but that might be even more awkward than "a or b or c or d or ... or x or y or z".  Thoughts?